specs/payment-api.yaml

```diff
@@ -683,6 +664,49 @@ paths:
         500:
           $ref: 'common.yaml#/responses/ServerError'
 
+  /payments/{paymentId}:
+    get:
+      tags:
+        - requestor
+        - provider
+      summary: Get Payment.
+      operationId: getPayment
+      parameters:
+        - $ref: '#/components/parameters/paymentId'
+      responses:
+        200:
+          $ref: '#/components/responses/Payment'
+        401:
+          $ref: 'common.yaml#/responses/Unauthorized'
+        404:
+          $ref: 'common.yaml#/responses/NotFound'
+        500:
+          $ref: 'common.yaml#/responses/ServerError'
+
+  /payments/status:
+    get:
+      tags:
+        - requestor
+        - provider
+      summary: Get status of the payment driver
+      description: >
+        This only relates to the erc20 driver, not erc20legacy. The returned
+        list contains individual status properties, which can be used to identify
+        problems like missing funds or misconfigured max fee per gas on a per-chain
+        (network) basis.
+      parameters:
+        - $ref: 'common.yaml#/parameters/network'
+        - $ref: 'common.yaml#/parameters/driver'
+      responses:
+        200:
+          $ref: '#/components/responses/DriverStatus'
+        400:
+          $ref: 'common.yaml#/responses/BadRequest'
```

Comment:
I would add 404 here. As we discussed @kamirr we want to return bad request only in case of malformated params but not existsting network/driver should be 404.